.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:  # https://docs.djangoproject.com/en/5.0/faq/install/#faq-python-version-support
         django-version: [50]
-        python-version: ["3.10", 3.11, 3.12]
+        python-version: [3.10, 3.11, 3.12]
```

Comment:
Are you sure these are unneeded? We've had issues with yaml treating 3.10 as 3.1 before.

Reply:
Not 100% sure, but I checked other projects that don't use double quote and it's running fine,

Reply:
I just set CI running for this PR, and it's trying to run against 3.1, so it looks like this is needed...

I'd suggest adding the quotes around all python versions here, for consistency.

Reply:
Weird, I added the quotes :+1: